librosa/filters.py

```diff
@@ -207,6 +207,10 @@ def mel(sr, n_fft, n_mels=128, fmin=0.0, fmax=None, htk=False,
         (area normalization).  Otherwise, leave all the triangles aiming for
         a peak value of 1.0
 
+    dtype : np.dtype
+        The data type of the output basis.
+        By default, uses 32-bit floating point.
```

Comment:
By default, uses a single-precision floating-point format, i.e. np.float32.
A double-precision format corresponds to dtype=np.float64